bump_definitions.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Axis Communications AB.
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Creates new versions of existing type definitions by increasing the
+desired version position (major, minor, or patch) and creating a new
+matching file.
+"""
+
+import argparse
+import fnmatch
+import re
+import sys
+import time
+from pathlib import Path
+from typing import List
+
+import semver
+
+import versions
+
+
+def _bump_versions(root: Path, position: str, pattern: str) -> List[Path]:
+    """Find definition files in the given directory, match their types
+    against the given glob pnattern, and create new files with version
+    bumps in the selected position. Returns the paths of the created
+    files.
+    """
+    result = []
+    for typename, version in versions.latest_in_dir(root).items():
+        if not fnmatch.fnmatch(typename, pattern):
+            continue
+
+        old_path = root / typename / f"{version}.yml"
+        new_version = getattr(version, "bump_" + position)()
+        new_path = old_path.with_name(f"{new_version}.yml")
+        new_path.write_text(
+            _transform_definition(old_path.read_text(encoding="utf-8"), new_version),
+            encoding="utf-8",
+        )
+        result.append(new_path)
+    return result
+
+
+def _transform_definition(
+    old_definition: str, version: semver.version.Version, copyright_year: int = None
+) -> str:
+    """Return an updated type definition with the new version and
+    copyright year patched in.
```

Comment:
What about the 'and others' statement that we've used when there is more than one company involved? Have we said that we should/could skip that from now on? See for example https://github.com/eiffel-community/eiffel/blob/master/definitions/EiffelActivityTriggeredEvent/2.0.0.yml#L1

Reply:
I don't recall any discussion about that. I deliberately left it out from this script since all current files have copyright notices that end with "and others", i.e. I don't know what even should be done.

Reply:
From the beginning the copyright statement didn't say 'and others' before someone from another company updated the file. That can be seen in several old .md files for example. But it shouldn't be any problem to have the copyright statement say 'and others' even if only one person or company has been involved. We did have discussions about this, but that might have been before the formal TC was formed.